### ISSUE TITLE
Incorrect language when more than one summary

### ISSUE THREAD

**Issue by mattgarrish** [w3c/publ-a11y#527]
The epub algorithm catches the first accessibility summary, but the xpath for the language finds the last instance of any summary or an ancestor with an xml:lang. So when you have two summaries, the first is output but if the second has a different xml:lang, it will be output with the second one's language.

I don't know if you can have more than one summary in an onix record, but if you can, it should be checked there, too.

**Comment by mattgarrish** [comment]
This may be an xpath 1.0 limitation when I look at it. I might have to modify the suggestion I made in #503
